src/i18n/DE/calendar-resources.ts

```diff
@@ -0,0 +1,34 @@
+export interface ICalendarResourceStrings {
+    igx_calendar_previous_month: string;
+    igx_calendar_next_month: string;
+    igx_calendar_select_month: string;
+    igx_calendar_select_year: string;
+    igx_calendar_range_start: string;
+    igx_calendar_range_end: string;
+    igx_calendar_selected_month_is: string;
+    igx_calendar_first_picker_of: string;
+    igx_calendar_multi_selection: string;
+    igx_calendar_range_selection: string;
+    igx_calendar_single_selection: string;
+    igx_calendar_singular_multi_selection: string;
+    igx_calendar_singular_range_selection: string;
+    igx_calendar_singular_single_selection: string;
+   }
+   
+   export const CalendarResourceStringsDE: ICalendarResourceStrings = {
+    igx_calendar_previous_month: 'Vorheriger Monat',
+    igx_calendar_next_month: 'Nächster Monat',
+    igx_calendar_select_month: 'Wähle Monat',
+    igx_calendar_select_year: 'Wähle Jahr',
+    igx_calendar_range_start: 'Datumsspanne Anfang',
+    igx_calendar_range_end: 'Datumsspanne Ende',
+    igx_calendar_selected_month_is: 'Der ausgewählter Monat ist ',
+    igx_calendar_first_picker_of: 'Der Erste Wähler von {0} beginnt von',
+    igx_calendar_multi_selection: 'Mehrfachauswahl Kalender mit {0} Datumwähler',
```

Comment:
Mehrfachauswahl-Kalender mit {0} Datumswähler(n)
(the (n) describes the optional plural)